plugins/mcreator-core/blockly/js/mcreator_extensions.js

```diff
@@ -137,6 +137,38 @@ Blockly.Extensions.register('min_max_fields_validator',
         });
     });
 
+// Mutator to disable the "biome filter" placement inside the "inline placed feature" block
+Blockly.Extensions.registerMixin('disable_biome_filter_inside_inline_placed_feature',

```

Comment:
Maybe in the future, there will be other blocks not working here, so a better name may be 

"disable_inside_inline_placed_feature"

only, same with translation text.